infrastructure/resource_group_level/modules/function-acbs.bicep

```diff
@@ -83,6 +84,7 @@ resource functionAcbs 'Microsoft.Web/sites@2022-09-01' = {
   tags: {}
   kind: 'functionapp,linux,container'
   properties: {
+    enabled: enabled
```

Comment:
Note that when deployed this produces the Azure Function App in the `disabled` state. (Not `stopped`).
It isn't clear if the individual functions might still run. They are set to `enabled` though I hope that the overall App being disabled prevents them executing. 

I suspect this means that one can't enable it in the UI and will need to redeploy!

There doesn't seem an obvious way to enable / Start it in the portal. Weirdly, you can click "Stop" and it reports success, but the UI still shows the Stop button, not Start. 

I'm not really sure how to test this. Looking at the metrics, the memory working set is at zero while disabled, so it looks like it is inactive!

Is all this ok?